app/views/user_mailer/authentication.text.erb

```diff
@@ -0,0 +1,3 @@
+Here's your sign in link:
+
+<%= link_to nil, new_session_url(auth_token: @authentication.token) %>
```

Comment:
Would something like this work too?

```suggestion
<%= new_session_url(auth_token: @authentication.token) %>
```

Reply:
I think so, since there can't be any link elements anyway.